cs-maps/src/main/java/io/cloudslang/content/maps/utils/SortMaps.java

```diff
@@ -0,0 +1,78 @@
+package io.cloudslang.content.maps.utils;
+
+import java.util.Comparator;
+
+public class SortMaps implements Comparator<String> {
+    private final boolean isDigit(char ch) {
+        return ((ch >= 48) && (ch <= 57));
+    }
+
+    private final String getChunk(String s, int slength, int marker) {
+        StringBuilder chunk = new StringBuilder();
+        char c = s.charAt(marker);
+        chunk.append(c);
+        marker++;
+        if (isDigit(c)) {
+            while (marker < slength) {
+                c = s.charAt(marker);
+                if (!isDigit(c))
+                    break;
+                chunk.append(c);
+                marker++;
+            }
+        } else {
+            while (marker < slength) {
+                c = s.charAt(marker);
+                if (isDigit(c))
+                    break;
+                chunk.append(c);
+                marker++;
+            }
+        }
+        return chunk.toString();
+    }
+
+    public int compare(String s1, String s2) {
+        if ((s1 == null) || (s2 == null)) {
+            return 0;
+        }
```

Comment:
This does not mean that is s1 or s2 is null the comparator will consider them equals?